test/powershell/Modules/Microsoft.PowerShell.Utility/Tee-Object.Tests.ps1

```diff
@@ -18,7 +18,28 @@ Describe "Tee-Object" -Tags "CI" {
 	        Write-Output teeobjecttest3  | Tee-Object $teefile
 	        Get-Content $teefile | Should -BeExactly "teeobjecttest3"
 	        Remove-Item $teefile -ErrorAction SilentlyContinue
-	    }
+        }
+
+        It "Should tee output to file using selected encoding" {
+            # Pass Encoding to Tee-Object
+            # and then read the resulting object back
+            # using the selected encoding
+            ForEach($encoding in @(
+                "ascii",
```

Comment:
Please remove obvious comments - we already have the `It` title.
Also please use `-TestCase` parameter instead of `ForEach`. (See examples in other tests).

Reply:
This is fixed.